package.json

```diff
@@ -6,7 +6,7 @@
 	"devDependencies": {
 		"git-rev-sync": "^1.8.0",
 		"happen": "^0.3.1",
-		"jake": "~0.5.16",
```

Comment:
Why does jake version change?

Reply:
Honestly, I don't know. IMHO jake itself did the change. I will revert this back.